_pages/home.md

```diff
@@ -1,6 +0,0 @@
----
-title: Home
```

Comment:
## Rename: Home -> About
For some reason I feel like for this site, "About" is a better title for the home page. I suppose because I think we want to keep the site lite, and there is no need for a separate home/about page. Basically what a home page is good for would be an expanded 'about' section (not needed), navigation (which is already handled by the nav bar), or calls to action / login / registration (we don't have / need that).